app/models/country_band.rb

```diff
@@ -1,48 +1,14 @@
 # frozen_string_literal: true
 
 class CountryBand < ApplicationRecord
-  BANDS = {
-    0 => {
-      value: 0.00,
-    },
-    1 => {
-      value: 0.19,
-    },
-    2 => {
-      value: 0.32,
-    },
-    3 => {
-      value: 0.45,
-    },
-    4 => {
-      value: 2.28,
-    },
-    5 => {
-      value: 3.00,
-    },
-  }.freeze
-
-  # According to WCA's current dues policy, the due amount per competitor is equivalent
-  # to this percent of registration fee. Only used if this due amount per competitor is
-  # larger than the due amount per competitor calculated from the competition's country band.
-  PERCENT_REGISTRATION_FEE_USED_FOR_DUE_AMOUNT = 0.15
-
-  def self.percent_registration_fee_used_for_due_amount(country_band)
-    return 0 if country_band.nil?
-    if country_band >= 3
-      0.15
-    elsif country_band >= 1
-      0.05
-    else
-      0.00
-    end
-  end
-
   belongs_to :country, foreign_key: :iso2, primary_key: :iso2
   validates_inclusion_of :iso2, in: Country::WCA_COUNTRY_ISO_CODES
-  validates_inclusion_of :number, in: BANDS.keys.freeze
```

Comment:
Can you hard-code the range from 0 to 5 instead of just completely deleting the line?
I know hard-coding is not very pretty, but effectively it was already hard-coded before (through an intermediate step which was getting the keys from the hard-coded hash)

Reply:
Okay done.